src/psychrometrics/t_mrt.js

```diff
@@ -0,0 +1,139 @@
+import { valid_range, round } from "../utilities/utilities.js";
+
+const g = 9.81;
+const c_to_k = 273.15;
+const cp_air = 1004;
+const mu = 0.0000181; // Pa s
+const k_air = 0.02662; // W/m-K
+const beta = 0.0034; // 1/K
+const nu = 0.0000148; // m2/s
+const alpha = 0.00002591; // m2/s
+const pr = (cp_air * mu) / k_air; // Prandtl constants, number
+const o = 0.0000000567;
+
+/**
+ * Converts globe temperature reading into mean radiant temperature in accordance with either the Mixed Convection developed by Teitelbaum E. et al. (2022) or the ISO 7726:1998 Standard [5]_.
```

Comment:
also please add the link to the corresponding function here, see `t_o` and `t_o_array`
make sure the `@see` is in a new line, otherwise our docs seem to not work